tests/system.bats

```diff
@@ -58,18 +58,6 @@ DOCKSAL_IP=192.168.64.100
 	unset output
 }
 
-@test "DSN: .docksal name resolution via nslookup" {
```

Comment:
unneeded, since this test was defunct and there is another nslookup test at the end of this file

Reply:
This one tests nslookup on the host. It's beneficial when running tests locally. I'll leave it there.